[2025-05-20T17:33:13.460+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monitor_spark_logs.check_spark_logs scheduled__2025-05-20T17:20:00+00:00 [queued]>
[2025-05-20T17:33:13.476+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monitor_spark_logs.check_spark_logs scheduled__2025-05-20T17:20:00+00:00 [queued]>
[2025-05-20T17:33:13.477+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-05-20T17:33:13.504+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): check_spark_logs> on 2025-05-20 17:20:00+00:00
[2025-05-20T17:33:13.513+0000] {standard_task_runner.py:57} INFO - Started process 230 to run task
[2025-05-20T17:33:13.522+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'monitor_spark_logs', 'check_spark_logs', 'scheduled__2025-05-20T17:20:00+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/monitor_spark.py', '--cfg-path', '/tmp/tmpg82qd7of']
[2025-05-20T17:33:13.530+0000] {standard_task_runner.py:85} INFO - Job 58: Subtask check_spark_logs
[2025-05-20T17:33:13.651+0000] {task_command.py:415} INFO - Running <TaskInstance: monitor_spark_logs.check_spark_logs scheduled__2025-05-20T17:20:00+00:00 [running]> on host 6bb1f84c11d3
[2025-05-20T17:33:13.796+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monitor_spark_logs' AIRFLOW_CTX_TASK_ID='check_spark_logs' AIRFLOW_CTX_EXECUTION_DATE='2025-05-20T17:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-20T17:20:00+00:00'
[2025-05-20T17:33:13.799+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-20T17:33:13.801+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n        docker exec gold_price_project-spark-1 bash -c \'\n        if grep -Ei "error|exception|traceback" /scripts/logs/spark.log; then\n            echo "❌ Erreur détectée dans spark.log"\n            curl -X POST -H "Content-Type: application/json"             -d "{\\"content\\":\\"❌ Erreur détectée dans spark.log ! Vérifiez le pipeline à {$(date +"%Y-%m-%d %H:%M:%S")}.\\"}"             https://discord.com/api/webhooks/1374382502981931008/BRaqnl_BHNxEep-dItvMinlhhcd9Tk5-sj3RbKdG3qDzLjwfYGRze8IZTxRoaiQj0DE2\n        else\n            echo "✅ Aucun problème détecté dans spark.log"\n        fi\n        \'\n        ']
[2025-05-20T17:33:13.829+0000] {subprocess.py:86} INFO - Output:
[2025-05-20T17:33:14.256+0000] {subprocess.py:93} INFO - ✅ Aucun problème détecté dans spark.log
[2025-05-20T17:33:14.271+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-20T17:33:14.310+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=monitor_spark_logs, task_id=check_spark_logs, execution_date=20250520T172000, start_date=20250520T173313, end_date=20250520T173314
[2025-05-20T17:33:14.344+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-20T17:33:14.372+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
